R/nm-file.R

```diff
@@ -87,20 +88,32 @@ nm_par_tab <- function(.mod) {
 
 #' @describeIn nm_file Reads the input data file from a `bbi_nonmem_model` or
 #'   `bbi_nonmem_summary` object
+#' @param filter Logical (`T`/`F`). If `TRUE`, filter data based on `IGNORE LIST`
+#'  or `ACCEPT LIST` options defined in the `$DATA` record.
 #' @importFrom data.table fread
 #' @importFrom tibble as_tibble
 #' @export
-nm_data <- function(.mod) {
-  check_model_object(.mod, c(NM_MOD_CLASS, NM_SUM_CLASS, NMSIM_MOD_CLASS))
+nm_data <- function(.mod, filter = FALSE) {
+  check_model_object(.mod, c(NM_MOD_CLASS, NM_SUM_CLASS, NMSIM_MOD_CLASS, NMBOOT_MOD_CLASS))
   .path <- get_data_path(.mod)
   verbose_msg(glue("Reading data file: {basename(.path)}"))
   .d <- fread(.path, na.strings = ".", verbose = FALSE)
   .d <- remove_dup_cols(.d)
   .d <- as_tibble(.d)
+
   names(.d) <- toupper(names(.d))
+
+  if(isTRUE(filter)){
```

Comment:
```r
Reading data file: acop.csv
  Filtered: 97.5% of records retained
  rows: 779
  cols: 10
```

how about

```r
Reading data file: acop.csv
  rows: 779 of 790
  cols: 10
```

or less like

```r
Reading data file: acop.csv
  rows: 779 (97.5%)
  cols: 10
```

Reply:
Went with your first recommendation in [8ad89b9](https://github.com/metrumresearchgroup/bbr/pull/707/commits/8ad89b90acdcb6a6086b68ee82e86b2d5273c975)